models/get-articles.model.js

```diff
@@ -15,4 +15,24 @@ function getArticleIdModel(article_id) {
         });
 }
 
-module.exports = getArticleIdModel
\ No newline at end of file
+function getArticlesModel() {
+    // comment_count which is the total count of all 
+    // the comments with this article_id - you should make
+    // use of queries to the database in order to achieve this.
+    
+    //  the articles should be sorted by date in descending order.
+    return db.query(`SELECT DISTINCT articles.article_id, articles.author, articles.title, articles.topic, articles.created_at, 
+    articles.votes, articles.article_img_url, (SELECT count(*) from comments WHERE comments.article_id = articles.article_id) as comment_count
+     FROM articles LEFT JOIN comments ON articles.article_id = comments.article_id ORDER BY articles.created_at DESC;`)
```

Comment:
Good work here with using an aggregate function (COUNT) to do the counting of total comments for you, well done!